hudi-client/src/main/java/org/apache/hudi/io/HoodieCommitArchiveLog.java

```diff
@@ -268,6 +270,19 @@ public Path getArchiveFilePath() {
     return archiveFilePath;
   }
 
+  private void writeHeaderBlock(Schema wrapperSchema, List<HoodieInstant> instants) throws Exception {
+    if (!instants.isEmpty()) {
+      Collections.sort(instants, HoodieInstant.COMPARATOR);
```

Comment:
It is unclear what order the HoodieInstant.COMPARATOR sorts the hoodie instants to, can we put a comment here ?

Reply:
ok, will do